docs/content/guides/getting-started/setting-options.md

```diff
@@ -663,13 +752,27 @@ hot.getCellMeta(0, 0).readOnly;
 // returns `true`
 hot.getCellMeta(1, 1).readOnly;
 ```
+:::
+
+::: only-for react
+```jsx
```

Comment:
This snippet is in a different section than the example before it, so these `readOnly` readings are kind of out of context. I would not remove the instance declaration above, but I'll leave the decision up to you.

Reply:
I'm unresolving this thread as there were no changes/comments here.

Reply:
my decision was to leave it as it is

Reply:
Continued in https://github.com/handsontable/handsontable/pull/9787#pullrequestreview-1085269238